packages/crayons-core/src/components/form/form.tsx

```diff
@@ -104,6 +104,12 @@ export class Form {
    */
   @Prop() customTypeMapper: any = {};
 
+  /**
+   * To check if hidden fields must be removed from DOM.
+   * Defaults to false.
+   */
+  @Prop({ mutable: true }) removeElementFromDomOnHide = false;
```

Comment:
Let's not have this as part of the main set of props. Let the method alone take this as argument.

Reply:
As discussed, lets only support hidden fields with `display:none`.